pkgs/by-name/no/notion-app/info.json

```diff
@@ -0,0 +1,14 @@
+{
+  "x86_64-darwin": {
+    "version": "4.2.0",
+    "url": "https://desktop-release.notion-static.com/Notion-4.2.0.zip",
+    "sha512": "FLptPNEtS9fTevSeGC00hDtpgSks+8JtEKRTtWlYPtI0vpA1KqixBdv2OaNSK1W7Krlsl25RpTOl8cJdQxcv4Q==",
+    "releaseDate": "2024-12-11T23:55:52.967Z"
+  },
+  "aarch64-darwin": {
+    "version": "4.2.0",
+    "url": "https://desktop-release.notion-static.com/Notion-arm64-4.2.0.zip",
+    "sha512": "cxfO3Bm7ZzAQMi0Pdwd3nvQlRPjn4w7j0ojYUCcn660YsBtoVkpNhiuqg9pbWzY0Umh+/8Zig9CGXKjjP94Iww==",
```

Comment:
I think this is very close to a SRI hash (the kind of stuff that is passed to the hash argument in fetchers). Maybe just prefixing the hash type would work. It would make things simpler.